include/z64camera.h

```diff
@@ -241,8 +332,8 @@ typedef enum {
     /* 0x1B */ CAM_DATA_MAX
 } CameraDataType;
 
-#define CAM_FUNCDATA_FLAGS(flags) \
-    { flags, CAM_DATA_FLAGS }
+#define CAM_FUNCDATA_INTERFACE_FIELD(flags) \
+    { flags, CAM_DATA_INTERFACE_FIELD }
 
```

Comment:
is the arg supposed to be flags still?
(ive not kept up with this PR as much, so im not fully up to speed on all the terms being used for things)